policyengine_canada/variables/gov/provinces/yt/tax/income/credits/childrens_fitness_tax_credit/yt_childrens_fitness_tax_credit.py

```diff
@@ -0,0 +1,24 @@
+from policyengine_canada.model_api import *
+
+
+class yt_childrens_fitness_tax_credit(Variable):
+    value_type = float
+    entity = Household
+    label = "Yukon children fitness tax credit"
+    definition_period = YEAR
+    defined_for = ProvinceCode.YT
+
+    def formula(household, period, parameters):
+        total_children = household("yt_cftc_eligible_children", period)
+        person = household.members
+        fees = person("child_physical_activities_fees", period)
+        disability_supplement = household(
+            "yt_childrens_fitness_tax_credit_disability_supplement", period
+        )
+        p = parameters(
+            period
+        ).gov.provinces.yt.tax.income.credits.childrens_fitness_tax_credit
+        return (
```

Comment:
Break this into multiple statements to make the code more comprehensible, e.g. give the two `min_` args names